### ISSUE TITLE
Nami Wallet crashes when signing tx due to a buggy code

### ISSUE THREAD

**Issue by kk-kien** [Berry-Pool/nami-wallet#446]
## Summary

A buggy code added 1.5 months ago is causing a crash when signing transactions.

https://github.com/Berry-Pool/nami-wallet/commit/93ca652be99006aa6b989fc63a5d0e474bc00e60#diff-93684c25afd055e453b1f64801526758960d648b2cf3fba1c2e4615a1b5ab927R407-R412

## Details

This piece of code does not seem to perform anything, but it is causing a crash when signing a tx.
```
Loader.Cardano.TransactionUnspentOutput.new()
            .output()
            .amount()
            .coin()
            .compare();
```

`Loader.Cardano.TransactionUnspentOutput.new` requires two arguments, but this code provides zero. Therefore, the app crashes.

<img width="1212" alt="image" src="https://user-images.githubusercontent.com/101089509/157430122-c42bb0f5-e6a5-41dd-928e-ca96a3e83a33.png">

**Please remove this buggy code**

-------

## Appendix

fyi, here is the console error when Nami Wallet crashes:
![image](https://user-images.githubusercontent.com/101089509/157430431-09b8d841-095d-4019-b281-8c1f55864674.png)

**Comment by kk-kien** [comment]
@alessandrokonrad

**Comment by alessandrokonrad** [comment]
Thanks! Totally forgot to remove that line. Was probably some testing I did there...

**Comment by kc97ble** [comment]
> Thanks! Totally forgot to remove that line. Was probably some testing I did there...

Thanks for the prompt response.

This issue is blocking my development. I hope that the new version will be released soon.

As a side note, I believe that this issue would have been easily prevented if the repo uses TypeScript or a similar type-checking system. One possible way to prevent similar issues is to migrate to TypeScript. This migration process can be done incrementally and therefore does not hurt the development.

(btw, @kk-kien and me are the same person)

**Comment by alessandrokonrad** [comment]
Yes fixed the issue already. Just have to wait until chrome store approves it.

**Comment by sixfeetunder7** [comment]
alessandro can you answer to our issues cause we have money inside nami ?

**Comment by sixfeetunder7** [comment]
@alessandrokonrad

**Comment by kk-kien** [comment]
Fixed in 3.1.4
